scripts/tests/e2/spells.lua

```diff
@@ -136,3 +136,16 @@ function test_familiar_lynx()
     assert_equal(1, u:get_skill('magic'))
     assert_equal(1, u:get_skill('perception'))
 end
+
+function test_bug_2480()
```

Comment:
Du kannst nicht garantieren, dass dieser Test immer klappt, und das Monster die Einheit tötet. Im Kampf ist ja schließlich Glück/Zufall drin. Selbst ein Test, der nur 1% false positives hat, ist Mist. Statistisch braucht man nicht einmal sehr viele davon, bevor man sich auf das Ergebnis eines vollen Testlaufes kaum noch verlassen kann.

Bessere Lösung: Die Berechnung der Magieresistenz aus terminate() in eine neue Funktion extrahieren, und für diese einen Unit-Test schreiben.

Reply:
So, habe terminate mal so umgeschrieben, dass ich fast verstehen kann, was da alles passiert.

Soll ich den lua-test jetzt zur Dokumentation beibehalten oder löschen?